argcomplete/_check_module.py

```diff
@@ -12,25 +12,32 @@
 
     def find_spec(name):
         """Minimal implementation as required by `find`."""
-        f, path, _ = find_module(name)
+        try:
+            f, path, _ = find_module(name)
+        except ImportError:
+            return None
         has_location = path is not None
         if f is None:
             return ModuleSpec(None, has_location, [path])
         f.close()
         return ModuleSpec(path, has_location, None)
 
 
+class LocateFailure(RuntimeError):
+    pass
+
+
 def find(name):
     names = name.split('.')
     spec = find_spec(names[0])
-    if not spec.has_location:
-        raise Exception('cannot locate file')
+    if spec is None or not spec.has_location:
+        raise LocateFailure('cannot locate file')
```

Comment:
It would make for nicer debugging if we split out these two cases - `None` means no module with that name was found, whereas `has_location` being `False` means a module was found but it does not have a corresponding file on disk.